### PR TITLE
Add new "customerSupport" field to BookingInterface

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
I'm adding new `customerSupport` field for booking. Currently, it contains only `guaranteeChat`, but I created this because more CS related fields are expected to land in upcoming months with new features.

```
{
  customerBookings {
    edges {
      node {
        customerSupport {
          hasGuaranteeChat
        }
      }
    }
  }
}
```

### CODE REVIEW

File: src/booking/types/outputs/customerSupport/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import { GraphQLObjectType, GraphQLBoolean } from 'graphql';
+
+import guaranteeChat from './guaranteeChat';
+
+export default new GraphQLObjectType({
+  name: 'BookingCustomerSupport',
+  fields: {
+    guaranteeChat: {
```
Comment (michalsanger) [inline src/booking/types/outputs/customerSupport/index.js]: Bool props should be prefixed, I suggest `hasGuaranteeChat` naming

File: src/booking/types/outputs/customerSupport/__tests__/guaranteeChat.test.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
+// @flow
+
+import MockDate from 'mockdate';
+
+import guaranteeChat from '../guaranteeChat';
+import { createLeg, hourInMillis } from '../../../../services/testUtils';
+
+describe('Services.guaranteeChat', () => {
+  beforeEach(() => MockDate.set(hourInMillis * 24));
```
Comment (michalsanger) [inline src/booking/types/outputs/customerSupport/__tests__/guaranteeChat.test.js]: All these `hourInMillis` operations should be replaced with https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#instance-method-plus

Comment (michalsanger) [reply src/booking/types/outputs/customerSupport/__tests__/guaranteeChat.test.js]: Don't multiple `hourInMillis` but use `DateTime.local().plus({ days: 1 })`

Comment (jaroslav-kubicek) [reply src/booking/types/outputs/customerSupport/__tests__/guaranteeChat.test.js]: @michalsanger fixed 💦

**Pull request merged**